kernel/src/process/wait.rs

```diff
@@ -96,17 +116,93 @@ pub fn wait_child_exit(
     Ok(zombie_child)
 }
 
+pub enum WaitStatus {
+    Zombie(Arc<Process>),
+    Stop(Arc<Process>, SigNum),
+    Continue(Arc<Process>),
+}
+
+impl WaitStatus {
+    pub fn pid(&self) -> u32 {
+        self.process().pid()
+    }
+
+    pub fn exit_status(&self) -> u32 {
```

Comment:
Nowhere in the man pages (https://man7.org/linux/man-pages/man2/wait3.2.html) does it mention the term "exit status." I don't know where it comes from.

Perhaps the `exit_status_ptr` in `sys_wait4` should be renamed as `status_ptr`, and this method is renamed as `fn status_code(self) -> u32`.